internal/datanode/channel_meta.go

```diff
@@ -114,7 +118,15 @@ type ChannelMeta struct {
 	segMu    sync.RWMutex
 	segments map[UniqueID]*Segment
 
-	needToSync   *atomic.Bool
+	// isHighMemory is intended to trigger the syncing of segments
+	// when segment's buffer consumes a significant amount of memory.
+	isHighMemory *atomic.Bool
+
+	// flushTs is intended to trigger:
+	// 1. the syncing of segments when consumed ts exceeds flushTs;
+	// 2. the updating of channelCP when channelCP exceeds flushTs.
+	flushTs *atomic.Uint64
```

Comment:
So its a minTs for channel to flush all streaming data <= flushTs into S3, which means to trigger a sync when bufferred segments ts >= flushTs, and update cp if buffered channelCP.ts reaches flustTs, took me a while to understands the meaning of the param.

Perhaps rename it with syncThreshold, means any segments whose buffer exceeds the thresholds with be synced

and set a const NoThreshold = math.MaxInt64, cuz math.MaxInt64 do make people wonder why when reading codes

Reply:
setSyncThreshold(ts)

cleanSyncThreahold/resetSyncThreshold or ....

would be betther to replace setFlushts